contentcuration/contentcuration/frontend/channelEdit/components/edit/EditView.vue

```diff
@@ -73,7 +73,12 @@
               <VAlert v-else-if="!areDetailsValid" :value="true" type="error" outline icon="error">
                 {{ $tr('errorBannerText') }}
               </VAlert>
-              <DetailsTabView :key="nodeIds.join('-')" :nodeIds="nodeIds" />
+              <DetailsTabView
+                :key="nodeIds.join('-')"
+                ref="detailsTab"
+                :nodeIds="nodeIds"
+                :closingModal="closingModal"
```

Comment:
This doesn't seem to be a prop on DetailsTabView any more, so I think this line is vestigial and can be removed.